geonetwork/transform-mcp-iso19115-3/schema/iso19115-3/convert/ISO19139/mapping/DQ.xsl

```diff
@@ -108,26 +108,32 @@
     <!--
     gmd:lineage moves directly under MD_Metadata
     -->
-    <xsl:for-each select="gmd:DQ_DataQuality/gmd:lineage/gmd:LI_Lineage">
+    <xsl:for-each select="gmd:DQ_DataQuality">
       <!--
       gmd:DataQuality objects without lineage go to ISO 19157
       -->
       <xsl:element name="mdb:resourceLineage">
```

Comment:
Are you saying that it is already creating dataQuality elements that do not have a lineage element or that there is potential for it to do so? The test contains an empty lineage element that gets carried across. Should it not do that? I'm just not a fan of testing for empty tags

Reply:
It will create empty lineage tags if there are only scope and report elements in the data quality element (lineage is optional).  No idea whether we have this situation, but we don't want to do this. Its not testing for empty tags - its testing for a lineage element, i.e. testing for when it should create a lineage element.  This is what we want to do.

Reply:
I see, I added the filter as requested and added an additional test element to the source data that should not be carried across as it is missing a lineage element. Test passes.